ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -453,6 +453,12 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(lastValue).to(equal("2"))
 			}
+
+			it("should not keep resulting signal alive indefinitely") {
+				weak var signal: Signal<AnyObject, NoError>? = Signal.never.map { $0 }
```

Comment:
I'd be surprised if this fails! This is even more reduced than I think, good catch :)

Could you write another, similar one that actually subscribes and then disposes the subscription?
That way we verify that fixing this also makes that other test pass :)